src/reconstruction/refinement/updateMetNames.m

```diff
@@ -0,0 +1,30 @@
+function [updatedModel] = updateMetNames(originalModel, modelToUpdate)
+% Updates model.metNames in a new model, using metNames of an original
+% model.
+%
+% USAGE:
+%
+%    [updatedModel] = updateMetNames(originalModel, modelToUpdate)
+%
+% INPUTS:
+%    originalModel:      COBRA model structure with correct model.metNames
+%    modelToUpdate:      COBRA model structure that needs to have its
+%                        model.metNames updated 
+%
+% OUTPUT:
+%    updatedModel:       COBRA model structure with corrected metNames
+%
+% .. Authors:
+%       - written by Diana El Assal 27/06/2017
+
+indMets = findMetIDs(originalModel, modelToUpdate.mets);
+updatedModel = modelToUpdate;
+updatedModel.metNames = [];
```

Comment:
Is it intentional, that every existing metName gets removed? 
It could be useful to e.g. only update those names, where the "reference" model has a value, and leave the others as they are, (and only create the metNames field if it does not exist).
Also: metNames should be a cell array {} not a double array [] and empty entries should be initialized with '' and not [].